src/qtiCommonRenderer/renderers/interactions/ExtendedTextInteraction.js

```diff
@@ -709,7 +719,7 @@ function inputLimiter(interaction) {
                 // if plain text - then limit input right after composition end event
```

Comment:
TODO: maxlength handling changed in https://github.com/oat-sa/tao-item-runner-qti-fe/pull/430, need to recheck that.

[_And why does that PR call `e.preventDefault()` on `compositionend`? I highly doubt it's a good thing. It call same handler on `blur`, but how does that help?_]